data/helpers.d/utils

```diff
@@ -24,3 +24,62 @@ ynh_get_plain_key() {
         fi
     done
 }
+
+# Restore a previous backup if the upgrade process failed
+#
+# usage:
+# ynh_backup_before_upgrade
+# ynh_clean_setup () {
+# 	ynh_backup_after_failed_upgrade
+# }
+# ynh_abort_if_errors
+#
+ynh_backup_after_failed_upgrade () {
```

Comment:
So, in this case, to follow the real function of this helper.
May we should rename it ynh_restore_backup_if_upgrade_failed